relayer/message_relayer.go

```diff
@@ -64,7 +64,7 @@ func newMessageRelayer(
 	quorum, err := relayer.globalConfig.GetWarpQuorum(destinationBlockchainID)
 	if err != nil {
 		relayer.logger.Error(
-			"Failed to get warp quorum",
+			"Failed to get warp quorum from config. Relayer may not be configured to deliver to the destination chain.",
```

Comment:
can you change the ordering between zap.Error and zap.String though?

Reply:
> nit I don't think we need the extra error log changes, since this shouldn't happen now due to destination chain not being configured.

I'm going to keep the exapanded error log, since it's possible that would change in the future.